### PR TITLE
Adds MapAssert.hasEntrySatisfying assertion

### PR DESCRIPTION
**Pull request by mchmielarz**
#56 hasEntrySatisfying part

### CODE REVIEW

File: src/main/java/org/assertj/vavr/api/AbstractMapAssert.java [hunk 1]
```diff
@@ -69,6 +70,22 @@ public SELF allSatisfy(BiConsumer<? super KEY, ? super VALUE> entryRequirements)
...
+    /**
+     * Verifies that the actual map contains a value for the given {@code key} that satisfies the given {@code valueCondition}.
+     *
+     * @param key            the given key to check.
+     * @param valueCondition the given condition for check value.
+     * @return {@code this} assertion object.
+     * @throws NullPointerException if the given values is {@code null}.
+     * @throws AssertionError       if the actual map is {@code null}.
+     * @throws AssertionError       if the actual map not contains the given {@code key}.
+     * @throws AssertionError       if the actual map contains the given key, but value not match the given {@code valueCondition}.
```
Comment (Opalo) [inline src/main/java/org/assertj/vavr/api/AbstractMapAssert.java]: _value does not match_?

File: src/main/java/org/assertj/vavr/internal/Maps.java [hunk 2]
```diff
@@ -35,16 +39,42 @@
...
+     * @param <K>            key type
+     * @param <V>            value type
+     * @param info           contains information about the assertion.
+     * @param actual         the given {@code Map}.
+     * @param key            the given key to check.
+     * @param valueCondition the given condition for check value.
+     * @throws NullPointerException if the given values is {@code null}.
+     * @throws AssertionError       if the actual map is {@code null}.
+     * @throws AssertionError       if the actual map not contains the given {@code key}.
+     * @throws AssertionError       if the actual map contains the given key, but value not match the given {@code valueCondition}.
```
Comment (Opalo) [inline src/main/java/org/assertj/vavr/internal/Maps.java]: Ditto.

File: src/main/java/org/assertj/vavr/internal/Maps.java [hunk 3]
```diff
@@ -35,16 +39,42 @@
...
+     */
+    @SuppressWarnings("unchecked")
+    public <K, V> void assertHasEntrySatisfying(AssertionInfo info, Map<K, V> actual, K key,
+                                                Condition<? super V> valueCondition) {
+        assertContainsKeys(info, actual, key);
+        conditions.assertIsNotNull(valueCondition);
```
Comment (Opalo) [inline src/main/java/org/assertj/vavr/internal/Maps.java]: Maybe it's better idea to make this assertion first? It's fast and it fails you don't need to verify the keys.

Comment (mchmielarz) [reply src/main/java/org/assertj/vavr/internal/Maps.java]: Good idea

**Comment by mchmielarz** [comment]
Fixes #56

**Pull request merged**